roles/bind-keycloak-apb/templates/client_bearer.json.j2

```diff
@@ -1,7 +1,7 @@
 {
   "id": "{{ CLIENT_ID }}",
-  "clientId": "{{ CLIENT_NAME }}-{{ CLIENT_TYPE }}",
-  "name": "{{ CLIENT_NAME }}",
+  "clientId": "{{ CLIENT_ID }}-{{ CLIENT_TYPE }}",
+  "name": "{{ CLIENT_ID }}",
```

Comment:
@JameelB change seems consistent with that in mobile-cli which is used as a filter,[ see recent change](https://github.com/aerogear/mobile-cli/blob/master/pkg/cmd/clientConfig.go#L76) so LGTM